terraform/deployments/concourse-iam/main.tf

```diff
@@ -27,7 +27,7 @@ provider "aws" {
 }
 
 resource "aws_iam_role" "govuk_concourse_deployer" {
-  name        = "govuk-concourse-deployer"
+  name        = "${terraform.workspace == "default" ? "govuk-concourse-deployer" : "govuk-concourse-deployer-${terraform.workspace}"}"
```

Comment:
I don't think this one is needed - the `concourse-iam` deployment is separate to the rest of govuk. It's just here to give the concourse worker permission to run the terraform. I don't think that needs to be workspace aware.

Reply:
Fixed